packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -1573,6 +1573,20 @@ describe('KeyringController', () => {
         expect(cancelSignRequestStub.called).toBe(true);
       });
     });
+
+    describe('cancelQRSync', () => {
+      it("should call qr keyring's method", async () => {
+        await setupQRKeyring();
+        const qrKeyring =
+          await signProcessKeyringController.getOrAddQRKeyring();
+
+        const cancelSyncRequestStub = sinon.stub(qrKeyring, 'cancelSync');
+        cancelSyncRequestStub.resolves();
+        // eslint-disable-next-line node/no-sync
+        await signProcessKeyringController.cancelQRSync();
```

Comment:
During our discussion this morning, we thought that `cancelQRSync` would have been the name that matched _most_ of the QR related methods, but I'm doing a couple more considerations now.

We don't have control over QRKeyring's method name, but we can perhaps avoid the `Sync` method name suffix, as violates our eslint rules (so we will probably have to eslint-disable this call also in clients) - moreover, this method is not really sync as it is a Promise, so might be confusing

Perhaps this new method can be called `cancelSyncQR`, so we can remove this eslint exception at least

Reply:
As an alternative, we could call it `cancelQRSynchronization`, I think either of the two work

Reply:
I think `cancelQRSynchronization` as a name makes sense to me.